packages/client/src/store/modules/grants.js

```diff
@@ -1,5 +1,19 @@
 const fetchApi = require('@/helpers/fetchApi');
 
+// fields with ⚠️ are not yet implemented in the api
```

Comment:
The ones without the warning _should_ work, but I haven't tested them all again in this new branch.